pipeline_steps/maas.groovy

```diff
@@ -51,11 +50,11 @@ def verify(vm=null) {
   ) //conditionalStage
 }
 
-List get_maas_token_and_url(String username, String api_key, String region, String venv) {
+List get_maas_token_and_url(String username, String api_key, String region) {
   def token_url = sh (
     script: """#!/bin/bash
 cd ${env.WORKSPACE}/rpc-gating/scripts
-. ${venv}/bin/activate
+. ${env.WORKSPACE}/.venv/bin/activate
 ./get_maas_token_and_url.py --username ${username} --api-key ${api_key} --region ${region}
 """,
     returnStdout: true,
```

Comment:
What about the use of a venv in entity_cleanup? Why does that one not need changing?

GitHub won't let me comment on the specific line in question.